test/parallel/test-http-same-map.js

```diff
@@ -0,0 +1,56 @@
+// Flags: --allow_natives_syntax
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const host = common.localhostIPv4;
+const port = common.PORT;
```

Comment:
I wouldn't stop something from landing over it, but yes, I would say that there is a preference for `server.listen(0)` over `server.listen(common.PORT)` where possible.